apps/desktop/desktop/src/main/java/bisq/desktop/components/controls/MaterialTextField.java

```diff
@@ -529,7 +560,9 @@ protected double getFieldLayoutY() {
     protected double computeMinHeight(double width) {
         if (helpLabel.isManaged()) {
             return helpLabel.getLayoutY() + helpLabel.getHeight();
-        } else {
+        } else if(errorLabel.isManaged()){
+            return errorLabel.getLayoutY() + errorLabel.getHeight();
+        }else{
```

Comment:
This was not addressed either:

> Best to check the reformat checkbox at the commit interface in IntelliJ so that code is correctly formatted (just a missing space where you add the errorLabel)